app/views/govuk_publishing_components/components/_select.html.erb

```diff
@@ -37,6 +48,10 @@
       <%= label_tag(id, label, class: label_classes) %>
     <% end %>
 
+    <% if hint_text %>
+      <%= tag.span hint_text, id: hint_id, class: "govuk-hint" %>
```

Comment:
just to note that[ in govuk-frontend v3.8 `span` became `div`](https://github.com/alphagov/govuk-frontend/issues/1835) and in v4 styles which made the hint <span> display as block [were removed](https://github.com/alphagov/govuk-frontend/releases/tag/v4.0.0#:~:text=Update%20the%20HTML%20for%20hints)

migration can either be addressed here or in https://github.com/alphagov/govuk_publishing_components/pull/2533

Reply:
The [hint component already uses a `div` element](https://github.com/alphagov/govuk_publishing_components/blob/master/app/views/govuk_publishing_components/components/_hint.html.erb). However, we have two instances (character count and checkboxes) where the component is not being used and a `span` is still in place. I have updated #2533 to address these two special cases (until we update them to use the hint component). For this PR I suggest we use the hint component [as we do for most form components](https://github.com/alphagov/govuk_publishing_components/search?q=components%2Fhint).

Reply:
Thanks for the input both. Yes, agree that we should definitely use the hint component here, have changed 🤦

Reply:
> The [hint component already uses a `div` element](https://github.com/alphagov/govuk_publishing_components/blob/master/app/views/govuk_publishing_components/components/_hint.html.erb). However, we have two instances (character count and checkboxes) where the component is not being used and a `span` is still in place. I have updated #2533 to address these two special cases (until we update them to use the hint component). For this PR I suggest we use the hint component [as we do for most form components](https://github.com/alphagov/govuk_publishing_components/search?q=components%2Fhint).

sorry din't see you folk have a hint component 👍

Reply:
Thanks for the nudge, @kr8n3r – otherwise I wouldn't have found the two exceptions 😄